ui/src/app.tsx

```diff
@@ -43,12 +43,43 @@ export default function App() {
             throw new Error(`No UI configuration in underlay ${name}`);
           }
 
+          const uiConfig = JSON.parse(underlay.uiConfiguration);
+
+          uiConfig.cohortReviewConfig.pages = [
+            {
+              type: "occurrenceTable",
+              id: "condition",
+              title: "Condition",
+              plugin: {
+                occurrence: "condition_occurrence",
+                columns: [
+                  { key: "condition", width: "100%", title: "Condition name" },
+                  { key: "start_date", width: 200, title: "Start date" },
+                  { key: "end_date", width: 200, title: "End date" },
+                ],
+              },
+            },
+            {
+              type: "occurrenceTable",
+              id: "procedure",
+              title: "Procedure",
+              plugin: {
+                occurrence: "procedure_occurrence",
+                columns: [
+                  { key: "procedure", width: "100%", title: "Procedure name" },
+                  { key: "date", width: 200, title: "Date" },
+                ],
+              },
+            },
+          ];
+
           return {
             name: underlay.name,
             displayName: underlay.displayName ?? underlay.name,
             primaryEntity: underlay.primaryEntity,
             entities: entitiesRes.entities,
-            uiConfiguration: JSON.parse(underlay.uiConfiguration),
+            //uiConfiguration: JSON.parse(underlay.uiConfiguration),
```

Comment:
Delete?

Reply:
Oops, done. I didn't mean to send this quite yet. I've moved the UI config changes into the underlays and re-expanded them.